### ISSUE TITLE
Feature Request: Per tracker seed ratio

### ISSUE THREAD

**Issue by neo-neo1** [transmission/transmission#1461]
Currently transmission has options to set seed ratio globally or individually per torrent. It would be very useful to have a seed ratio per tracker setting.

**Why?**

- Different trackers require different seed ratios, 1:1 for private trackers and less for public trackers.

- Media managers such as SickChill/Sonarr/Radarr add and start the torrent automatically without user input (link via RPC).

**Comment by xavery** [comment]
> 1:1 for private trackers and less for public trackers.
Interesting.

**Comment by neo-neo1** [comment]
> > 1:1 for private trackers and less for public trackers.
> 
> Interesting.

It's a minimum rule on most private trackers, or at least the one's I have access to. Just an example.

**Comment by svein83** [comment]
The specific ratios you prefer aren't important in this thread. A feature to auto-set ratio based on tracker would make life way easier. Some torrents should be seeded forever, and some only for a while.

**Comment by neo-neo1** [comment]
> The specific ratios you prefer aren't important in this thread. A feature to auto-set ratio based on tracker would make life way easier. Some torrents should be seeded forever, and some only for a while.

@svein83 Reread my post, pay attention to the word "example" when referring to my "specific ratios". Your second sentence reiterates exactly what my original post describes.

**Comment by svein83** [comment]
Yeah. I totally agree with you. That's why I gave your feature request a +1.

My comment was only an attempt to support and clarify the suggestion.

**Comment by moetayuko** [comment]
Each torrent already has an `isPrivate` property, so I guess this is somewhat straightforward to implement. Having two global seed ratios for public and private should be sufficient.
